cookbooks/omnitruck/recipes/_application.rb

```diff
@@ -1,113 +0,0 @@
-include_recipe 'runit::default'
-
-directory '/srv/omnitruck/shared' do
-  owner 'omnitruck'
-  group 'omnitruck'
-end
-
-directory '/srv/omnitruck/shared/pids' do
-  owner 'omnitruck'
-  group 'omnitruck'
-end
-
-# Poller configuration
-poller_path = "/srv/omnitruck/shared/poller_data/"
-poller_cache_path = "#{poller_path}/cache"
-
-directory poller_path do
-  owner 'omnitruck'
-  group 'omnitruck'
-end
-
-directory poller_cache_path do
-  owner 'omnitruck'
-  group 'omnitruck'
-end
-
-release = data_bag_item('omnitruck', node['applications']['omnitruck'])
-
-artifact_deploy 'omnitruck' do
-  version release['version']
-  artifact_location release['artifact_location']
-  deploy_to '/srv/omnitruck'
-  owner 'omnitruck'
-  group 'omnitruck'
-  action :deploy
-
-  symlinks(
-    'unicorn.rb' => 'config/unicorn.rb',
-    'poller_config.yml' => 'config/config.yml',
-    'poller_data/cache' => 'release-metadata-cache'
-  )
-
-  before_symlink Proc.new {
-    # Omnitruck webapp configuration
-    unicorn_config "/srv/omnitruck/shared/unicorn.rb" do
-      listen '/tmp/.omnitruck.sock.0' => { :backlog => 1024, :tcp_nodelay => true }
-      worker_processes 8
-      owner 'omnitruck'
-      group 'omnitruck'
-      mode  '0755'
-    end
-
-    template "/srv/omnitruck/shared/poller_config.yml" do
```

Comment:
Where is this config rendered at now? I saw `cookbooks/omnitruck/templates/default/poller_config.yml.erb` was removed from the repo also.